guide/yaml/yaml-reference.md

```diff
@@ -58,7 +58,7 @@ the entity being defined, with these being the most common:
     and optionally a `period`, to create a sensor feed which populates the sensor with
     the given name by running the given command (on an entity which as an ssh-able machine)
 
-  * `org.apache.brooklyn.core.sensor.windows.WinRmCommandSensor`: For command supplied via WinRm. Takes a `name` and `command`,
+  * `org.apache.brooklyn.core.sensor.windows.WinRmCommandSensor`: For command supplied via WinRm. Takes a `name`, `command`, `executionDir`,
```

Comment:
Seems `executionDir` is required right now, otherwise it will error out - worth mentioning it here. LGTM otherwise.